File: Week 1 C/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    int spaces;
    int row = 1;
    
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    
    for (int i = h; i > 0; i--) 
    {
        spaces = i - 1;
        while (spaces > 0)
        {
            printf(" ");
            spaces--;
        }
        for (int j = 0; j < row; j++) 
        {
            printf("#");
        }
        row++;
        printf("\n");
    }
}

File: Week 6 Python/pset6/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

row = 1
for i in range(h, 0, -1):
    # print("i", i)
    spaces = i - 1
    while (spaces > 0):
        print(" ", end="")
        spaces -= 1

    for j in range(row):
        print("#", end="")

    row += 1

    print()
